### PR TITLE
Feature/bander

### PR DESCRIPTION
**Pull request by thuanheo**
banner & service

### CODE REVIEW

File: baitap2.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="stylebaitap2.css">
+</head>
+<body>
+    <div class="header">
+        </div>
+    <div class=banner>
+        <div class="back">
+            <a class="load_img" href="">
```
Comment (NgoVanTien) [inline baitap2.html]: Use href = "#" without the URL

File: baitap2.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                <img class="position_img"src="ic_back_header.png" alt="" width="30" height="30">
```
Comment (NgoVanTien) [inline baitap2.html]: Space before src=

**Comment by NgoVanTien** [comment]
It's great :+1:  Có 2 commit nhé.

**Comment by thuanheo** [comment]
em sua lai roi :v

**Pull request merged**